ci/README.md

```diff
@@ -1 +1,120 @@
-This is a WIP conversion of internal/mage CI targets to `dagger do` commands. It's not invoked by our CI yet.
+# CI
```

Comment:
Something strongly worth noting here is that the version of the CLI you use to call this module is extremely important. Ideally it should be the same version that's in our GHA yaml that invokes all this.

Fortunately this is quite easy since it's always the most recent stable release, so should just be our CLI install instructions. But still worth noting because if you forget to upgrade your host's CLI you may hit unexpected errors or other weird behavior.

Reply:
Done!